app/main/forms.py

```diff
@@ -1854,6 +1854,7 @@ class PDFUploadForm(StripWhitespaceForm):
         validators=[
             FileAllowed(["pdf"], "Save your letter as a PDF and try again."),
             DataRequired(message="You need to choose a file to upload"),
+            FileSize(max_size=(2 * 1024 * 1024), message="File must be smaller than 2MB"),
```

Comment:
Technically this is Mebibytes (MiB) rather than Megabytes (MB) so this could be changed to be more consistent with the other FileSize validator.

```suggestion
            FileSize(max_size=(2e6), message="File must be smaller than 2MB"),
```